### PR TITLE
Changed in-text driver name to "Vulkan"

### LINKED ISSUE #36124: Vulkan: Create new project window has option to use GLES 3.0
**Godot version:**
Master

**OS/device including version:**
Linux Mint 19.3

**Issue description:**
image
Of course on creation this project will be Vulkan
image

**Steps to reproduce:**
Hit create new project

**Minimal reproduction project:**

### PR DESCRIPTION
**Pull request by Teashrock**
But it still requires a change of "driver_name" in backend itself.
This also fixes #36124 .

### CODE REVIEW

File: editor/project_manager.cpp [hunk 1]
```diff
@@ -900,7 +900,7 @@ class ProjectDialog : public ConfirmationDialog {
 		rs_button->set_meta("driver_name", "GLES2");
 		rvb->add_child(rs_button);
 		l = memnew(Label);
-		l->set_text(TTR("- Lower visual quality\n- Some features not available\n- Works on most hardware\n- Recommended for web games"));
+		l->set_text(TTR("- Lower visual quality\n- Obsolete and less accurate API, which produces not so fast code\n- Some features not available\n- Works on most hardware\n- Recommended for web games"));
```
Comment (akien-mga) [inline editor/project_manager.cpp]: I don't think this is a useful addition. If it's "obsolete, less accurate, not fast", why would we even propose it to users?

Comment (Teashrock) [reply editor/project_manager.cpp]: Okay, you have reason. But I just wanted to emphasize one more notable difference between them. How I should write it correctly? Or maybe delete at all, if not needed.

**Review by Teashrock — COMMENTED** [review]
Better now?

**Comment by Teashrock** [comment]
Uuuuh... Sometimes I'm getting in the interface of GitHub.

File: editor/project_manager.cpp (hunk 1 above)
Comment (Calinou) [reply editor/project_manager.cpp]: I think you can keep the current description and replace "All features available" with "Some features not implemented yet".

Comment (Teashrock) [reply editor/project_manager.cpp]: Better now.

**Comment by aaronfranke** [comment]
@Teashrock See the [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html) article if you're having trouble with Git. A PR like this should generally be one commit, so you'd need to squash and force-push.

**Comment by Teashrock** [comment]
@aaronfranke 
Did read that guide already. And I suppose, that I need to fixupinstead of squash.

**Comment by Teashrock** [comment]
All done.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**